website/index.html

```diff
@@ -861,33 +861,23 @@
         </a>
       </div>
     </div>
-    <!--<div class="row platinum">-->
-      <!--<div class="col-sm-8 offset-sm-2">-->
-        <!--&lt;!&ndash; Nittany AI alliance &ndash;&gt;-->
-        <!--<a href="https://www.nittanyai.psu.edu/" target="_blank">-->
-          <!--<img src="assets/images/sponsors/nittanyai-night.svg" alt="Nittany AI Alliance"-->
-               <!--class="img-fluid nittany-ai-day">-->
-        <!--</a>-->
-      <!--</div>-->
-    <!--</div>-->
 
-    <!-- Gold Sponsors -->
-    <!--<div class="row gold" style="padding-top: 2%">-->
-      <!--<div class="col-sm-6">-->
-        <!--&lt;!&ndash; General Motors &ndash;&gt;-->
-        <!--<a href="https://www.gm.com/" target="_blank">-->
-          <!--<img src="assets/images/sponsors/GM-night.svg" alt="GM" class="img-fluid gm-day">-->
-        <!--</a>-->
-      <!--</div>-->
 
-      <!--<div class="col-sm-6">-->
-        <!--&lt;!&ndash; Oracle &ndash;&gt;-->
-        <!--<a href="https://oracle.com" target="_blank">-->
-          <!--<img src="assets/images/sponsors/Oracle-night.svg" alt="Oracle"-->
-               <!--class="img-fluid oracle-day">-->
-        <!--</a>-->
-      <!--</div>-->
-    <!--</div>-->
+    <!-- Gold Sponsors -->
+    <div class="row gold" style="padding-top: 2%">
+      <div class="col-sm-6">
+      <!-- Geico -->
+        <a href="https://www.geico.com/" target="_blank">
+          <img src="assets/images/sponsors/Geico-day.svg" alt="Vertex" class="img-fluid Geico-night">
+        </a>
+      </div>
+      <div class="col-sm-6">
+        <!-- GCP -->
+        <a href="https://www.geico.com/" target="_blank">
```

Comment:
Should this be going to the GCP website?

Reply:
done